### DISCUSSION TITLE
Filtering in DataView component PrimeReact

### DISCUSSION THREAD

**Discussion by d-szuffy** [primefaces/community#2115] [PrimeReact]
Hi. I couldn't find anything related nor there is any info in documentation. Is there any way to easily implement filtering to DataView component? It is very similar do DataTable, so I would be surprised if there wasn't.

**Comment by melloware** [comment]
I don't think DataView currently supports filtering.  It would be a feature request

**Comment by d-szuffy** [comment]
I see, thank you for reply. Gotta do it on my own then :)

**Comment by nimo23** [comment]
As all data related components (datatable, datatree,etc.) supports **filtering** out of the box, it would be good to provide filtering also for https://primereact.org/dataview/.
